server/sonar-web/src/main/webapp/WEB-INF/app/controllers/sessions_controller.rb

```diff
@@ -41,10 +41,7 @@ def login
     begin
       self.current_user = User.authenticate(params[:login], params[:password], servlet_request)
       if logged_in?
-        if params[:remember_me] == '1'
-          self.current_user.remember_me
-          cookies[:auth_token] = { :value => self.current_user.remember_token , :expires => self.current_user.remember_token_expires_at, :http_only => true }
-        end
+        set_user_session
```

Comment:
Because during authentication, rails is only feeding session[:user_id], not the Java UserSession. This last one is feed at the begin of each page (We need to remove rails authentication !).